websecurityscanner/setup.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A setup module for the GAPIC Web Security Scanner API library.
+
+See:
+https://packaging.python.org/en/latest/distributing.html
+https://github.com/pypa/sampleproject
+"""
+
+from setuptools import setup, find_packages
+import io
+import sys
+
+install_requires = [
+    'google-api-core >= 0.1.0, < 0.2.0dev',
```

Comment:
this need to just be `google-api-core[grpc]>=1.0.0,<2.0.0`, no need for the `googleapis-common-protos`.